cpp/arcticdb/version/version_map_entry.hpp

```diff
@@ -346,11 +358,14 @@ inline std::optional<VersionId> get_next_version_in_entry(const std::shared_ptr<
     return std::nullopt;
 }
 
-inline std::optional<AtomKey> find_index_key_for_version_id(VersionId version_id, const std::shared_ptr<VersionMapEntry>& entry) {
+inline std::optional<AtomKey> find_index_key_for_version_id(VersionId version_id, const std::shared_ptr<VersionMapEntry>& entry, bool included_deleted=true) {
```

Comment:
* This line is now too long. Please wrap
* `find_index_key_for_version_timestamp` below differs by just a few bytes. Please move it up so it immediately follows this function.

Reply:
done